packages/react-native-win32/.gitignore

```diff
@@ -1,12 +1,12 @@
 /dist
 /flow
-/flow-typed
 /index.*
+/interface.*
 /IntegrationTests
 /jest
 /lib
 /Libraries
 /RNTester
 /RNTester.*
 /temp
-/WorkingHeaders
```

Comment:
>WorkingHeaders [](start = 1, length = 14)

This has been gone long enough that I should be able to clean it up now without annoying everyone